sql/src/main/java/org/apache/druid/sql/http/ResultFormat.java

```diff
@@ -100,6 +100,20 @@ public Writer createFormatter(final OutputStream outputStream, final ObjectMappe
     {
       return new ObjectLinesWriter(outputStream, jsonMapper);
     }
+  },
+  
+  OBJECTARRAY {
```

Comment:
This should be added to the doc page `docs/querying/sql.md`. I don't think the name is very clear though. Maybe "wrappedObject" would be clearer in relation to the other ones?

- object: array of objects
- array: array of arrays
- wrappedObject: object containing an array of objects

Reply:
fixed